RELEASE.md

```diff
@@ -1,14 +1,59 @@
-# Stacks Release Process
+# Appsody Stacks Release Process
+A release in this repo represents a new version of a particular stack - denoted by the release tag. It will contain the following assets:
+- archives for all the templates for the released stack on [GitHub releases](https://github.com/appsody/stacks/releases)
+- latest indexes for each of the repos (`incubator`, `experimental`, `stable`) on [GitHub releases](https://github.com/appsody/stacks/releases)
+- updated docker hub images with appropriate tags on [dockerhub](https://hub.docker.com/u/appsody)
 
-This document outlines the process for building, testing and releasing Appsody stacks. 
+## Steps to create a new stack release
+The Appsody stacks are made available by creating a tagged GitHub release. Follow these steps to create a new release of an Appsody stack:
 
-`appsody/stacks` contains multiple stacks repos such as `experimental`, `incubator` and `stable`. Each repo includes several stacks. The build process allows you to build and release each stack and the indexes for each stack repo. 
+### Trigger the release on GitHub
+1. Navigate to https://github.com/appsody/stacks/releases
+1. Click _Draft a new release_ button
+1. Ensure the target branch is __master__
+1. Define a tag in the format `<stack-id>-v<version>` (example: java-microprpfile-v0.2.4)
+1. Set a title in the format `<stack-id> v<version>` (example: java-microprofile v0.2.4)
+1. Describe the notable changes to the stack as release notes
+1. Tick _This is a pre-release_ checkbox
+1. Click _Publish release_ button
 
-[Travis-CI](https://travis-ci.org) is configured to build and test the stacks and indexes on GitHub pull requests. When a GitHub release is created manually, the built and tested stacks and indexes are published on DockerHub and GitHub releases. 
+### Monitor the Travis build
+1. Watch the [Travis build](https://travis-ci.com/appsody/stacks) for the release and ensure it passes
+1. Check the release artefacts to ensure these all exist:
+    - archives for all the templates for the released stack on [GitHub releases](https://github.com/appsody/stacks/releases)
+    - latest indexes for each of the repos (`incubator`, `experimental`, `stable`) on [GitHub releases](https://github.com/appsody/stacks/releases)
+    - updated docker hub images with appropriate tags on [dockerhub](https://hub.docker.com/u/appsody)
 
-## Build
+### Update the pre-release
+1. Navigate to https://github.com/appsody/stacks/releases
+1. Click on the title of the pre-release you have just created
+1. Click _Edit release_ button
```

Comment:
nit: we could make this bold perhaps: **_Edit release_**

Reply:
done